### PR TITLE
Feature: Renamed Copy location to Copy path

### LINKED ISSUE #11945: Feature: Rename copy location to copy path
### What feature or improvement do you think would benefit Files?

Adapt the "Copy location" action to its specific function of "copy the path of the element".

Understanding that the location of a file or folder can be better interpreted as the place where it is located or stored, the "Copy location" function -from the popup menu that is displayed when right-clicking on a selected file or folder- should allow copying to the clipboard its location, that is, the "Folder/Directory that contains it" and not the "Path of the element" as it currently works.

Hence, it is proposed to replace the current "Copy location" string with the new "Copy path" or "Copy as path" that exactly expresses its function, that is, "Copy the path of the element" (File/Folder), as displays the "Item Path" section of the "Preview Pane".

Picture:
Folder >> FeedbackHub >> Location: ----> 	C:\Users\dracp\Documents 
Folder >> FeedbackHub >> Path: -------->	C:\Users\dracp\Documents\FeedbackHub = Pane Preview / Item Path

230402 F 2 4 60 0_Propuesta de modificación_Copy Path x Copy Location

### Requirements

- Change Source String : "Copy location" to "Copy path"
- Update references in the code as well (property names etc)

### Files Version

2.4.60.0

### Windows Version

W11 22H2 22621.1485

### Comments

_No response_

### PR DESCRIPTION
**Pull request by hishitetsu**
The command list has also been updated to the latest information.

**Resolved / Related Issues**
- [X] Were these changes approved in an issue or discussion with the project maintainers? In order to prevent extra work, feature requests and changes to the codebase must be approved before the pull request will be reviewed. This prevents extra work for the contributors and maintainers.
   Closes #11945 

**Validation**
How did you test these changes?
- [X] Did you build the app and test your changes?
- [ ] Did you check for accessibility? You can use Accessibility Insights for this.
- [X] Did you remove any strings from the en-us resource file?
   - [X] Did you search the solution to see if the string is still being used? 
- [ ] Did you implement any design changes to an existing feature?
   - [ ] Was this change approved?
- [ ] Are there any other steps that were used to validate these changes?

### CODE REVIEW

File: src/Files.App/Strings/en-US/Resources.resw [hunk 1]
```diff
@@ -120,8 +120,8 @@
   <data name="NavigationToolbarNewWindow.Label" xml:space="preserve">
     <value>New window</value>
   </data>
-  <data name="CopyLocation" xml:space="preserve">
-    <value>Copy location</value>
+  <data name="CopyAsPath" xml:space="preserve">
+    <value>Copy as path</value>
```
Comment (yaira2) [inline src/Files.App/Strings/en-US/Resources.resw:124]:
```suggestion
    <value>Copy path</value>
```

File: src/Files.App/Strings/en-US/Resources.resw [hunk 2]
```diff
@@ -120,8 +120,8 @@
   <data name="NavigationToolbarNewWindow.Label" xml:space="preserve">
     <value>New window</value>
   </data>
-  <data name="CopyLocation" xml:space="preserve">
-    <value>Copy location</value>
+  <data name="CopyAsPath" xml:space="preserve">
```
Comment (yaira2) [inline src/Files.App/Strings/en-US/Resources.resw:123]:
```suggestion
  <data name="CopyPath" xml:space="preserve">
```

File: src/Files.App/Strings/en-US/Resources.resw (hunk 1 above)
Comment (hishitetsu) [reply src/Files.App/Strings/en-US/Resources.resw:124]: File Explorer labels it as "Copy as path". Do we not need to match the label?
image

Comment (yaira2) [reply src/Files.App/Strings/en-US/Resources.resw:124]: It doesn't have to be the same, I'm not sure what the "as" is for.

Comment (hishitetsu) [reply src/Files.App/Strings/en-US/Resources.resw]: I changed it to "Copy path". Actually, I think this is easier to understand too.

**Review by yaira2 — APPROVED** [review]
LGTM

**Pull request merged**